src/ChunkData.cpp

```diff
@@ -571,6 +574,10 @@ void cChunkData::SetSkyLight(const NIBBLETYPE * a_Src)
 
 cChunkData::sChunkSection * cChunkData::Allocate(void)
 {
+	if (m_WorldMemoryCounter != nullptr)
```

Comment:
Its probably simpler to just create a memory counter object in the tests, rather than having nullptr objects all over the place.